### PR TITLE
Adding simple parameter set for VM and VMSS creation

### PR DESCRIPTION
**Pull request by bganapa**
## Description

Porting the simple parameter sets in azure cmdlets to azurestack version

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by bganapa** [comment]
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/80/

File: src/StackAdmin/AzureRM/AzureRM.psd1 [hunk 1]
```diff
@@ -51,9 +51,9 @@
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.6.0'; },
+    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.7.0'; },
```
Comment (markcowl) [inline src/StackAdmin/AzureRM/AzureRM.psd1]: Seems like we need a version change

Comment (bganapa) [reply src/StackAdmin/AzureRM/AzureRM.psd1]: changed

Comment (cormacpayne) [inline src/StackAdmin/AzureRM/AzureRM.psd1]: @bganapa should this match the version of Profile that we have Compute depend on below? (5.8.2)

Comment (bganapa) [reply src/StackAdmin/AzureRM/AzureRM.psd1]: Yes . Updated it.

**Comment by bganapa** [comment]
@cormacpayne This should be good to merge. Thanks!

**Pull request merged**